### DISCUSSION TITLE
Error on omnisharp_mono setup

### DISCUSSION THREAD

**Discussion by willswats** [mason-org/mason.nvim#970] [Q&A] [answered]
Hello, I'm using [LunvarVim](https://github.com/LunarVim/LunarVim) and I'm receiving an error when attempting to setup the `omnisharp_mono` LSP:

```bash
[lspconfig] Cannot access configuration for omnisharp_mono. Ensure this server i
s listed in `server_configurations.md` or added as a custom server.
```

This error appears after adding this code snippet to my `config.lua` (after following the documentation [here](https://github.com/williamboman/mason-lspconfig.nvim/blob/main/lua/mason-lspconfig/server_configurations/omnisharp/README.md)):

```lua
local lspconfig = require "lspconfig"
lspconfig.omnisharp_mono.setup {}
```

I presume that the error is appearing due to `omnisharp_mono` being a server added by `mason-lspconfig`, which doesn't exist for `nvim-lspconfig`. I'm on Fedora Silverblue, I have `dotnet-sdk-6.0` and [mono-devel](https://www.mono-project.com/download/stable/#download-lin-fedora) installed. Does anyone know how to fix this?

**Comment by williamboman** [comment, marked as answer]
Hey! `mason-lspconfig` needs to be installed, and more importantly set up before you set up lspconfig. Likely the lspconfig snippet you provided is executing before `mason-lspconfig` is set up.

**Reply by willswats** [reply]
Thank you for you help! I added the snippet under where `mason-lspconfig` is setup in the LunarVim source code, and it works.
